elements/map/helpers.ts

```diff
@@ -213,3 +215,11 @@ export function isTopoJSON(text: string): boolean {
     return false;
   }
 }
+
+export function registerProjection(
+  name: string,
+  projection: string | proj4.ProjectionDefinition
```

Comment:
Wouldn't it be a very nice convenience function to make this async and fetch the definition for the user from https://epsg.io/?format=json&q=? Like in https://openlayers.org/en/latest/examples/reprojection-by-code.html.

Reply:
i added this as another helper functionality, utilizing `fromEPSGCode`